src/test/java/org/apache/commons/lang3/EnumUtilsTest.java

```diff
@@ -29,6 +29,7 @@
 import java.util.EnumSet;
 import java.util.List;
 import java.util.Map;
+import java.util.HashMap;
```

Comment:
Please run `mvn` locally before pushing to catch build errors.

Reply:
Hi,
Thank you for the feedback. I've made the changes. Please let me know if there's anything else.